hr_payroll_cancel/test/update_payroll_workflow.yml

```diff
@@ -0,0 +1,8 @@
+-
+  Workflow Mexican Electronic Payroll
+-
+  !python {model: hr.payslip}: |
```

Comment:
https://github.com/OCA/maintainer-tools/blob/master/template/module/hooks.py

https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#installation-hooks

This os useful when you need a change after install the module